README.md

```diff
@@ -162,7 +165,7 @@ This will generate the binaries inside the `dist_mist` or `dist_wallet` folder.
 
 ##### platform
 
-Additional you can only build the windows, linux or mac binary by using the `platform` option:
+Additional you can only build the windows, linux, mac or all binary by using the `platform` option:
```

Comment:
Probably better reworded as "you can build for a specific platform or all platforms together using the `platform` option"